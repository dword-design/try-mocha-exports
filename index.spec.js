import { expect } from 'expect'
import { toMatchImageSnapshot } from 'expect-mocha-image-snapshot'
import toMatchSnapshot from 'expect-mocha-snapshot'
import sharp from 'sharp'

expect.extend({ toMatchSnapshot })
expect.extend({ toMatchImageSnapshot })

export default {
	foo() {
		expect('blabla').toMatchSnapshot(this)
	},
	async bar() {
		const img = await sharp({
			create: {
				background: { b: 0, g: 255, r: 0 },
				channels: 3,
				height: 48,
				width: 48,
			},
		})
			.png()
			.toBuffer()
		expect(img).toMatchImageSnapshot(this)
	},
}
