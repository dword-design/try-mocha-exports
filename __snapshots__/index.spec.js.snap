// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`foo 1`] = `"blabla"`;
